### PR TITLE
add new target to forc

### PR DESCRIPTION
**Pull request by sdankel**
## Checklist

- [ ] I have linked to any relevant issues.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation where relevant (API docs, the reference, and the Sway book).
   - [ ] If my change requires substantial documentation changes, I have [requested support from the DevRel team](https://github.com/FuelLabs/devrel-requests/issues/new/choose)
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added (or requested a maintainer to add) the necessary `Breaking*` or `New Feature` labels where relevant.
- [ ] I have done my best to ensure that my PR adheres to [the Fuel Labs Code Review Standards](https://github.com/FuelLabs/rfcs/blob/master/text/code-standards/external-contributors.md).
- [ ] I have requested a review from the relevant team or maintainers.

### CODE REVIEW

File: forc-plugins/forc-client/src/constants.rs [hunk 1]
```diff
@@ -15,6 +16,7 @@ pub const DEVNET_FAUCET_URL: &str = "https://faucet-devnet.fuel.network";
 pub const TESTNET_FAUCET_URL: &str = "https://faucet-testnet.fuel.network";
 
 pub const TESTNET_EXPLORER_URL: &str = "https://app.fuel.network";
```
Comment (kayagokalp) [inline forc-plugins/forc-client/src/constants.rs:18]: I think we can change this into:

```suggestion
pub const TESTNET_EXPLORER_URL: &str = "https://app-tesnet.fuel.network";
```

Comment (JoshuaBatty) [reply forc-plugins/forc-client/src/constants.rs:18]: There is a spelling mistake in your suggestion there @kayagokalp , should be `https://app-testnet.fuel.network` .... testnet was missing a 't'

File: forc-plugins/forc-client/src/lib.rs [hunk 2]
```diff
@@ -14,19 +14,27 @@ pub struct NodeTarget {
...
     #[clap(long, env = "FUEL_NODE_URL")]
     pub node_url: Option<String>,
+
     /// Use preset configurations for deploying to a specific target.
     ///
-    /// You can also use `--node-url` or `--testnet` to specify the Fuel node.
+    /// You can also use `--node-url`, `--testnet`, or `--mainnet` to specify the Fuel node.
     ///
-    /// Possible values are: [beta-1, beta-2, beta-3, beta-4, local]
+    /// Possible values are: [beta-1, beta-2, beta-3, beta-4, devnet, local, testnet, mainnet]
```
Comment (JoshuaBatty) [inline forc-plugins/forc-client/src/lib.rs:25]: Let's remove the beta's and devnet 

```suggestion
    /// Possible values are: [local, testnet, mainnet]
```

File: forc-plugins/forc-client/src/constants.rs (hunk 1 above)
Comment (JoshuaBatty) [reply forc-plugins/forc-client/src/constants.rs:18]: i added this in #6634

File: forc-plugins/forc-client/src/lib.rs (hunk 2 above)
Comment (JoshuaBatty) [reply forc-plugins/forc-client/src/lib.rs]: added in https://github.com/FuelLabs/sway/pull/6634

**Review by JoshuaBatty — APPROVED** [review]

**Review by alfiedotwtf — APPROVED** [review]

**Pull request merged**